docs/agent-http-api.md

```diff
@@ -93,6 +93,9 @@ its configuration is returned. The same configuration is returned for all the
 following calls, until the agent sends the full experiment result to the crater
 server.
 
+Agents should specify their capabilities as a comma-separated list in a query
+parameter like so: `/next-experiment?capabilities=linux,big-hard-drive`.
```

Comment:
The rest of the API accepts inputs in a JSON-encoded request body. Could we switch to that for consistency?

Reply:
This is implemented and documented [here](https://github.com/rust-lang/crater/pull/450/files#diff-f07d4ff42e3271b0adff034b4fd54011R80)